tests/testPolicy.py

```diff
@@ -111,19 +111,25 @@ def testMerge(self):
         self.assertEqual(b['a.b.d'], 3)
 
     def testOpenDefaultPolicy(self):
-        policy = Policy(defaultInitData=('daf_persistence', 'testPolicy.yaml', 'tests'))
+        policyFile = Policy.defaultPolicyFile('daf_persistence', 'testPolicy.yaml', 'tests')
+        policy = Policy(policyFile)
         self.assertEqual(policy['exposures.raw.template'], 'foo/bar/baz.fits.gz')
 
     def testDumpLoad(self):
         self.policy.dumpToFile('testDumpFile.yaml')
-        loadedPolicy = Policy(filePath='testDumpFile.yaml')
+        loadedPolicy = Policy('testDumpFile.yaml')
         self.assertEqual(self.policy, loadedPolicy)
         os.remove('testDumpFile.yaml')
 
     def testNonExistantPolicyAtPath(self):
-        self.assertRaises(IOError, Policy, filePath="c:/policy.yaml")
-        self.assertRaises(IOError, Policy, filePath="c:/policy.paf")
-        self.assertRaises(IOError, Policy, filePath="c:/policy")
+        # Test that loading paf & yaml files that do not exist raises an IOError.
+        self.assertRaises(IOError, Policy, "c:/nonExistantPath/policy.yaml")
```

Comment:
Please use
```python
with self.assertRaises(IOError):
    Policy("c:....yaml")
```
as it is much clearer how the command is constructed that triggers the exception.